### PR TITLE
feat: add variables for configuring real_ip module

### PR DESCRIPTION
**Pull request by theseion**
Add all three real_ip module configuration options for nginx as
environment variables.

Replaces #108.

I've taken the liberty of updating the original PR and adding what @theMiddleBlue requested.

### CODE REVIEW

File: v3-nginx/Dockerfile-alpine [hunk 1]
```diff
@@ -124,6 +124,9 @@ ENV ACCESSLOG=/var/log/nginx/access.log \
     MODSEC_UPLOAD_DIR=/tmp/modsecurity/upload \
     PORT=80 \
     NGINX_ALWAYS_TLS_REDIRECT=off \
+    SET_REAL_IP_FROM="127.0.0.1,10.10.0.0/24" \
```
Comment (fzipi) [inline v3-nginx/Dockerfile-alpine:127]: Why the `10.10.0.0/24` here?

Comment (fzipi) [inline v3-nginx/Dockerfile-alpine:127]: Why the `10.10.0.0/24` here?

Comment (theseion) [reply v3-nginx/Dockerfile-alpine:127]: Good catch! I had that in there for debugging.

**Review by fzipi — APPROVED** [review]
LGTM

**Comment by theseion** [comment]
@theMiddleBlue Can you add your review please?

**Comment by theMiddleBlue** [comment]
it's cool that you can add a list of trusted ip for multiple set_real_ip_from
LGTM, thanks!

**Pull request merged**